public/manifest.json

```diff
@@ -37,7 +37,6 @@
     "scripts": ["common.js", "background.js"],
     "persistent": true
   },
-  "web_accessible_resources": [
-    "oasis-xu-frame.html"
-    ]
+  "web_accessible_resources": ["oasis-xu-frame.html"],
+  "externally_connectable": { "ids": [] }
```

Comment:
What's a benefit of adding this entry when empty `ids` array is the same as unspecified `externally_connectable` manifest key (as long as this theory is true)?

Reply:
Empty `externally_connectable` would do the same. But I think this is easier to understand intuitively, without looking at documentation :D 
and I can't add a comment